### DISCUSSION TITLE
How to integrate a plugin? Docs are not clear about it.

### DISCUSSION THREAD

**Discussion by mrpmohiburrahman** [ldelossa/nvim-ide#129] [Q&A] [answered]
Thanks for the awesome repo.

My question is, if I want to add nvim-tree, instead of built-in Explorer, how should I do it?

I am using lazy package manager.

I have tried adding bufferline.nvim, but it's also not working.

Here is my nvim config https://github.com/mrpmohiburrahman/dotfilesOSX/tree/main/nvim/nvim-ide/nvim

I wish there was an example nvim config repo of some sort with the custom plugin implemented.

Especially for the screen shot mentioned readme.
![image](https://github.com/user-attachments/assets/4d1bf1b6-7ad9-4141-96aa-539cfbe9eeca)

**Comment by ldelossa** [comment, marked as answer]
Unfortunately, that's not supported at this time. 

It would take some development effort, but integrating plugins could be possible. Tho I imagine lots and craziness would occur.
